### PR TITLE
dashboard: disable tx submit unless an action is added

### PR DESCRIPTION
**Pull request by chrisgarvin**

### CODE REVIEW

**Comment by chrisgarvin** [comment]
also added a line break to fix the following styling:

image

**Comment by chrisgarvin** [comment]
PTAL

File: dashboard/src/features/transactions/components/New/New.jsx [hunk 1]
```diff
@@ -153,7 +159,7 @@ class Form extends React.Component {
                 <tr>
                   <td><input id='submit_action_submit' type='radio' {...submit_action} value='submit' checked={submit_action.value == 'submit'} /></td>
                   <td>
-                    <label htmlFor='submit_action_submit'>Submit transaction to blockchain</label>
+                    <label htmlFor='submit_action_submit'>Submit transaction to blockchain</label><br></br>
```
Comment (dominic) [inline dashboard/src/features/transactions/components/New/New.jsx]: For tags with no content, let's use `<br />` style tags. Also - can you put this on a new line between the two labels?

File: dashboard/src/features/transactions/components/New/New.jsx [hunk 2]
```diff
@@ -162,7 +168,7 @@ class Form extends React.Component {
                 <tr>
                   <td><input id='submit_action_generate' type='radio' {...submit_action} value='generate' checked={submit_action.value == 'generate'} /></td>
                   <td>
-                    <label htmlFor='submit_action_generate'>Allow additional actions</label>
+                    <label htmlFor='submit_action_generate'>Allow additional actions</label><br></br>
```
Comment (dominic) [inline dashboard/src/features/transactions/components/New/New.jsx]: Same as above.

File: dashboard/src/features/transactions/components/New/New.jsx (hunk 1 above)
Comment (chrisgarvin) [inline dashboard/src/features/transactions/components/New/New.jsx]: done

File: dashboard/src/features/transactions/components/New/New.jsx (hunk 2 above)
Comment (chrisgarvin) [inline dashboard/src/features/transactions/components/New/New.jsx]: and done 👍

**Comment by dominic** [comment]
LGTM

**Pull request merged**